client/src/components/History/Content/SelectedItems.test.js

```diff
@@ -64,6 +64,17 @@ describe("History SelectedItems", () => {
         expectSelectionDisabled();
     });
 
+    it("should discard (but not disable) the current selection on `reset`", async () => {
+        const numberOfExpectedItems = 3;
+        await selectSomeItemsManually(numberOfExpectedItems);
+        expect(slotProps.selectionSize).toBe(numberOfExpectedItems);
+
+        await resetSelection();
```

Comment:
The `await` here should have no effect, since the `resetSelection` just calls the `resetSelection` from the slot which does not `async` operation, right?

Reply:
Interestingly if I remove it the test doesn't pass because we need to wait for the next tick. I guess this might be Vue Test Utils magic which is calling nextTick under the hood and waiting for the change.

Reply:
Interesting, thanks for checking it out.